student.rb

```diff
@@ -2,9 +2,12 @@
 
 # Class Student inherited from Person
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
-    super(name, age, parent_permission)
+    super(age, name, parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
   end
 
   def play_hooky
```

Comment:
> - [ ] Good job @SnakyMz, In accordance with the project's requirements which can be found [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Make%20sure%20that%20when%20setting%20the%20classroom%20for%20a%20student%20it%20also%20adds%20it%20to%20the%20classrooms%27%20students.), when setting the classroom for a student it also adds it to the classrooms' students. I recommend creating a method to achieve this. This enhancement will be highly beneficial.👍
> 
> You can use the below as a guide:
> 
>  ```
> def classroom=(classroom)
>       @classroom = classroom
>       classroom.students.push(self) unless classroom.students.include?(self)
>   end
> 
> ```

Reply:
Thanks for the suggestion, I will make changes as suggested